docs/lti13Instructors.md

```diff
@@ -0,0 +1,60 @@
+# LTI 1.3 configuration for instructors
+
+LTI 1.3 requires a trust relationship between the Learning Management System (Canvas) and
```

Comment:
I know the current beta is only concerned about Canvas, but I'd imagine this will eventually work for other LMSs as well (Moodle, etc.), right? Would it make more sense to list Canvas as a restriction of the beta (or public preview as mentioned by @mwest1066 below) instead of referring to Canvas as the only LMS?